### DISCUSSION TITLE
Editing "open hours" and other POI information

### DISCUSSION THREAD

**Discussion by michaelkirk** [headwaymaps/headway#278] [Ideas]
As laid out in [a recent project update](https://about.maps.earth/posts/2023/05/more-details-for-the-places-youll-go), now that we're exposing more POI information in Headway, it seems like a natural step to facilitate editing that information for places that don't yet have it or where what they have is outdated.

![image](https://github.com/headwaymaps/headway/assets/217057/04bb3a84-efa2-4a02-b1f2-3dec8ffb36a7)

There are some [challenges](https://about.maps.earth/posts/2023/05/more-details-for-the-places-youll-go/#cant-you-just) with this laid out in the post. Currently, we simply link people to openstreetmap.org, but maybe we can do better.

**Comment by robertgzr** [comment]
+1 for surfacing POI data in headway, but building out editing support seems like quite the rabbit hole to get lost in :P

maybe having some kind of interface to create OSM notes, like how [StreetComplete](https://streetcomplete.app/) and I think Maps.ME do it, is already enough?

speaking as someone that does a bit of mapping myself, when I use an application like headway, I'm in the frame of mind of a user and don't like to be distracted by mapping hehe, but I use the note-creation feature of osmAnd quite heavily. during a mapping session i then go through and process all the notes i created for myself (if somebody else didn't get to them first)
